src/mixins/legal-api-mixin.ts

```diff
@@ -236,4 +236,21 @@ export default class LegalApiMixin extends Mixins(CommonMixin) {
       }
     })
   }
+
+  /** Checks if the specified business has any pending tasks.
+   * @param businessId the business identifier (aka entity inc no)
+   * @returns True if there are any non-NEW tasks, else False
+   */
+  async hasPendingTasks (businessId: string): Promise<boolean> {
+    return this.fetchTasks(businessId).then(response => {
+      const tasks = response?.data?.tasks || []
+      return tasks.some(task => {
+        if (task?.task?.filing?.header) {
+          if (task.task.filing.header.status !== FilingStatus.NEW) {
+            return true
+          }
+        }
```

Comment:
so would a task with FilingStatus of COMPLETED still be considered a pending task?  or is this not a valid scenario because the api would never return a completed task?

Reply:
Right (the latter): a completed task should not be returned by `GET tasks`.